src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/ChatWithCustomData-CSharp.Web.csproj.in

```diff
@@ -45,6 +45,8 @@
 #elif (UseQdrant)-->
     <PackageReference Include="Aspire.Qdrant.Client" Version="${TemplatePackageVersion_Aspire}" />
     <PackageReference Include="Microsoft.SemanticKernel.Connectors.Qdrant" Version="${TemplatePackageVersion_MicrosoftSemanticKernel_Preview}" />
+<!--#elif (UseLocalVectorStore)-->
```

Comment:
As a separate issue (future us), we could consider moving to Central Package Management, which would then let us drop a Directory.Packages.props file into the folder and eliminate package versions from the project files in the template.